// tbclient.Profile.ProfileReqIdl
syntax = "proto3";

import "CommonReq.proto";

message ProfileReqIdl {
    message DataReq {
        int64 uid = 1;
        uint32 need_post_count = 2;
        uint32 pn = 6;
        CommonReq common = 9;
        int32 page = 15;
    }
    DataReq data = 1;
}
